CHANGELOG.md

```diff
@@ -5,12 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.1.0]
```

Comment:
for me, this is the releasing process that will do this king of modifications.
for me, keep "Unreleased" here

Reply:
Ok, done